Mcst/Node.cs

```diff
@@ -45,10 +63,15 @@ public void Update(double result)
         _wins += result;
     }
 
-    public TMove GetBestMove()
+    public TMove GetBestMove(MctsVersion mctsVersion, IMcstGame<TMove> game)
     {
-        return Children.OrderByDescending(c => c.Visits).Select(c => c.MoveMade).FirstOrDefault()!;
-    }
+        if (Children.Count == 0 && mctsVersion == MctsVersion.Heuristic)
```

Comment:
Czy to jest potrzebne? Czy ta sytuacja jest wgl nożliwa?

Reply:
jest możliwa, gdy podczas porównywania 2 algorytmów zaczyna wersja z heurystyką - chce wtedy wybrać miejsce sąsiadujące z jakimś już zajętym polem, a jeszcze żadne pola nie są zajęte